css/style.css

```
@@ -24,6 +25,97 @@ body {
   color: var(--text-primary-color);
 }
 
+/* Header component */
+
+.navbar {
+  width: 100%;
+  height: 52px;
+  align-items: center;
+  display: flex;
+  justify-content: flex-end;
+  opacity: 1;
+  position: fixed;
+}
+
+/* navbar-hero is behind navbar-content until animation is done */
+
+.navbar-hero {
+  background-color: transparent;
+  padding: 0 8px 0 10px;
+  z-index: 4;
```

it's a good practice to keep z-index as low as possible
```suggestion
  z-index: 1;
```